perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -6934,7 +6959,7 @@ private void updateInverseDependenciesForAttribute(Map<AttributeDefinition, Set<
 					AttributeDefinition attrDef = getAttributeDefinition(sess, s);
 					strongDepSet.add(attrDef);
 				} catch (AttributeNotExistsException ex) {
-					log.error("For attribute dependency name " + s + "can't be found attributeDefinition.");
+					log.error("For attribute dependency name {} can't be found attributeDefinition.", s);
```

Comment:
I would prefer something like: ``AttributeDefinition can't be found for dependency {}.`` Same for the else branch.